### DISCUSSION TITLE
Tablet driver

### DISCUSSION THREAD

**Discussion by huoyan1231** [ppy/osu#22642] [Q&A]
### Type

Game behaviour

### Bug description

just open osu lazer

### Screenshots or videos

![image](https://user-images.githubusercontent.com/53113734/218707633-023ac958-f758-4364-a091-70fcf0f0ec42.png)

### Version

2023.207.0

### Logs

[performance-input.log](https://github.com/ppy/osu/files/10731374/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/10731375/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/10731376/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10731377/updater.log)
[database.log](https://github.com/ppy/osu/files/10731378/database.log)
[input.log](https://github.com/ppy/osu/files/10731379/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/10731380/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/10731382/network.log)
[performance.log](https://github.com/ppy/osu/files/10731384/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/10731385/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/10731386/performance-draw.log)

**Comment by huoyan1231** [comment]
![image](https://user-images.githubusercontent.com/53113734/218708107-5bf01eed-39e9-402d-9fa0-407ac0edb77b.png)

**Comment by mk56-spn** [comment]
Please *do not* open issues with nondescript comments such as " just open lazer " i can gauge its some sort of tablet issue  but i have this same model and it works fine so you're gonna have to make it a bit clearer what you are referring to.

Is it an issue with the area, input , etc....?

**Comment by huoyan1231** [comment]
> Please _do not_ open issues with nondescript comments such as " just open lazer " i can gauge its some sort of tablet issue but i have this same model and it works fine so you're gonna have to make it a bit clearer what you are referring to.
> 
> Is it an issue with the area, input , etc....?

just this popup 
no other issues

**Comment by mk56-spn** [comment]
input log says  "Failed to find auxiliary device, express keys may be unavailable", i assume keys here refers to buttons some tablets have, but the CTL 672 doesn't have buttons.  Probably going to be an issue coming from upstream  since OTD is used for lazer tablet integration. Can't repro on linux so might be platform specific as well
